src/sniffs/readme/class-parser.php

```diff
@@ -0,0 +1,817 @@
+<?php
+/**
+ * Theme Sniffer WP Theme readme.txt Parser
+ *
+ * Based upon the WordPress.org Plugin Readme Parser, which was
+ * based on Baikonur_ReadmeParser from https://github.com/rmccue/WordPress-Readme-Parser
+ *
+ * @package Theme_Sniffer\Sniffs\Readme
+ *
+ * @since   1.1.0
+ */
+
+// Ignoring this from sniffs for now since the majority was pulled from .org to retain compatibility.
+// phpcs:ignoreFile
```

Comment:
Not a big fan of this 😕 

Since this is an OS project, I'd like for it to present the best practices 🙂There don't seem to be many issues in this file that can be fixed